portfolio/src/main/webapp/index.html

```diff
@@ -30,68 +30,83 @@
+                id="working-hour-end"
+                name="working-hour-end"
+                value="17:00"
+                onchange="checkWorkHourRange()"
+              />
+              <small class="d-none" id="working-hour-warning"></small>
+            </li>
             <li class="nav-item dropdown">
               <a
                 class="nav-link dropdown-toggle"
-                href="#"
-                id="navbar-dropdown"
+                id="navbarDropdown"
                 role="button"
                 data-toggle="dropdown"
                 aria-haspopup="true"
                 aria-expanded="false"
               >
-                Scheduling Preferences
+                Scheduling algorithm preferences
               </a>
-              <div class="dropdown-menu" aria-labelledby="navbar-dropdown">
-                <a>Working Hours:</a>
-                <form class="px-4 py-3">
-                  <input
-                    type="time"
-                    id="working-hour-start"
-                    name="working-hour-start"
-                    class="form-control"
-                    value="09:00"
-                  />
-                  <span>to</span>
-                  <input type="time" id="working-hour-end" name="working-hour-end" class="form-control" value="17:00" />
-                </form>
-              </div>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown"></div>
             </li>
           </ul>
         </div>
       </nav>
+
       <div class="row">
         <div id="new-task-wrapper" class="col-md-12 col-lg-6 col-xl-3">
-          <h2>Add the tasks you would like to schedule for today</h2>
+          <h2>1. Add the tasks you would like to schedule for today</h2>
+          <h5>Fields with asterisk (*) are required.</h5>
           <form id="new-task-form">
             <div class="form-group">
-              <label for="new-task-name">Task name:</label>
-              <input type="text" id="new-task-name" class="form-control" name="new-task-name" placeholder="New Task" />
+              <label for="new-task-name">Task name*:</label>
+              <input type="text" id="new-task-name" class="form-control" name="new-task-name" placeholder="New Task" required="true" />
               <div class="invalid-feedback" id="empty-name-message">Name cannot be empty!</div>
             </div>
-
             <label for="new-task-description">Task description:</label><br />
             <textarea
               id="new-task-description"
               class="form-control"
               name="new-task-description"
               placeholder="Description"
-            ></textarea>
-            <br />
-
+            ></textarea
+            ><br />
+ 
             <div class="form-row">
               <div class="col">
-                <label for="new-task-estimated-length">Duration:</label>
+                <label for="new-task-estimated-length">Duration*:</label>
```

Comment:
You included an asterisk here but did not add `required="true"`.

Reply:
Updated.